sys-apps/tuned/tuned-2.13.0.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
why not 7?

Reply:
Because I copied the file and didn't think about it.